### DISCUSSION TITLE
Self-learning Courses

### DISCUSSION THREAD

**Discussion by erbaibhav** [github/india#564] [Students]
Hello Community!

I have started learning and completing the self-learning part...but in each of the courses after completing also it shows in progress..😔
What to do??

**Comment by RhugwedaShedge** [comment]
While completing the courses on MS learn, make sure you are logged in to the MS Learn account. Also, there is a scheduled refresh that happens at certain time intervals. Your progress gets updated after the refresh. So check if you have completed the course after a refresh and wait for the next refresh.
